### PR TITLE
Add plots and tables in consultation

### PR DESCRIPTION
**Pull request by divyagar**
#1576

### CODE REVIEW

**Comment by bodhish** [comment]
@divyagar can I merge it?

File: src/Components/Facility/ConsultationDetails.tsx [hunk 1]
```diff
@@ -464,14 +466,20 @@ export const ConsultationDetails = (props: any) => {
               ></ABGPlots>
             </div>
           )}
-          {tab === "NURSING" && (
+          {tab === "IO%20BALANCE" && (
```
Comment (bodhish) [inline src/Components/Facility/ConsultationDetails.tsx:469]:
```suggestion
          {tab === "IO_BALANCE" && (
```

**Review by bodhish — CHANGES_REQUESTED** [review]

File: src/Components/Facility/Consultations/IOBalancePlots.tsx [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+    },
+    legend: {
+      data: [
+        "Adrenalin",
+        "Dopamine",
+        "Nor-adrenalin",
+        "Vasopressin",
+        "Dobutamine",
+      ],
+    },
```
Comment (bodhish) [inline src/Components/Facility/Consultations/IOBalancePlots.tsx:87]: This data will be dynamic. We should convert this to a table. (Means more than the listed medicines can be added to infusions)

The 2 possible plots in IO balance are Total Intake and Total Output.

**Review by bodhish — CHANGES_REQUESTED** [review]

File: src/Components/Facility/Consultations/IOBalancePlots.tsx (hunk 2 above)
Comment (divyagar) [reply src/Components/Facility/Consultations/IOBalancePlots.tsx:87]: Okay. Meaning I have to remove all plots except `total intake` and `total output` and convert them into table ?

File: src/Components/Facility/ConsultationDetails.tsx (hunk 1 above)
Comment (divyagar) [reply src/Components/Facility/ConsultationDetails.tsx:469]: `IO_BALANCE` is not working because there is a space between words.

**Pull request closed without merging**